ee/codegen/src/__test__/nodes/__snapshots__/error-node.test.ts.snap

```diff
@@ -0,0 +1,46 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`ErrorNode > basic > getNodeDisplayFile 1`] = `
+"from vellum_ee.workflows.display.nodes import BaseErrorNodeDisplay
+from ...nodes.error_node import ErrorNode
+from uuid import UUID
+from vellum import VellumError
+from vellum_ee.workflows.display.nodes.types import PortDisplayOverrides
+from vellum_ee.workflows.display.vellum import NodeDisplayData, NodeDisplayPosition
+
+
+class ErrorNodeDisplay(BaseErrorNodeDisplay[ErrorNode]):
+    name = "error-node"
+    node_id = UUID("2cd960a3-cb8a-43ed-9e3f-f003fc480951")
+    label = "Error Node"
+    error_output_id = UUID("69250713-617d-42a4-9326-456c70d0ef20")
+    source_handle_id = UUID("e4dedb66-0638-4f0c-9941-6420bfe353b2")
+    target_handle_id = UUID("370d712d-3369-424e-bcf7-f4da1aef3928")
+    error_inputs_by_name = {
+        "error_source_input_id": VellumError(
+            message="Something went wrong!", code="USER_DEFINED_ERROR"
+        )
+    }
```

Comment:
[blocking] I wouldn't expect this to show up here. `error_inputs_by_name` isn't an attribute of `BaseErrorNodeDisplay`

Reply:
I included this in the display class to generate the inputs for the node. I couldn't find/don't know an existing way to grab the values for it

Reply:
Definitely had a hard time trying to take this piece out. It looks like it's pretty integrated into how we generate node_inputs in the serializer here: https://github.com/vellum-ai/vellum-python-sdks/blob/main/ee/vellum_ee/workflows/display/nodes/vellum/error_node.py#L30

Reply:
I’ll take a pass at this.

Reply:
Actually, let’s merge this without this change and Kira, please make me a ticket in our epic for me to try to clean up at a later time.

Reply:
Sounds like a plan!